Changelog.md

```diff
@@ -1,4 +1,3 @@
 # Change history
 
-See deployment.yml
```

Comment:
I liked the fact to have a Changelog like history file.
Was diffable and could be re-pushed to the release.
When I do typo, or for adding extra comment to a release.

I suppose that relying to https://github.com/docopt/docopts/releases removes the file based changelog behavior?

editing with vim, grepable, was parsable too. as a yaml file.

Reply:
Yeaaa, there's pros and cons to it for sure. I linked it in the `.goreleaser.yaml`, but here's [the docs](https://goreleaser.com/customization/changelog/) on how `goreleaser` creates the changelog by default -- it basically generates it automatically from git logs.

The changelog you have here is pretty new with only two releases, so I didn't think it'd be a big difference.

That being said, this is customizable (as everything in `goreleaser` is). With the [`--release-notes` flag](https://goreleaser.com/customization/release/#custom-release-notes), you can specify a file to use as release notes.

So I can make some modifications if you still want the file history in the repo. Will just have to parse out the specific version's notes.

Reply:
Ok, so I added a `CHANGELOG.md` with all the release notes that were in `deployment.yml`.
This is an actual, readable markdown file with syntax highlighting, unlike the quoted text in the YAML before. Users can read this directly too.

I added a tiny `awk` script to parse out the top-most heading and use that as `--release-notes` for `goreleaser`.
Also added `make test_release_notes` to verify the release notes.

And then updated the release docs to reflect this.